### DISCUSSION TITLE
Plaid support

### DISCUSSION THREAD

**Discussion by silentnyte** [firefly-iii/firefly-iii#7065] [General]
Howdy, thinking about switching to Firefly and trying to figure out how to get all my banks connected. I see the firefly-plaid-connector and I was just curious if there plans to integrate plaid support into the firefly data-importer?

https://github.com/dvankley/firefly-plaid-connector-2

**Comment by JC5** [comment]
There are no plans to do that.

**Comment by Archenoth** [comment]
> https://github.com/dvankley/firefly-plaid-connector-2

Oh dang, thank you for mentioning the existence of this~

I nearly made a feature request before finding this discussion since the current two services don't support my bank, but Plaid does!

**Comment by cskujawa** [comment]
I implemented a similar connector using Plaid as well, I opted to do it myself because I haven't worked with Java in a long time and I like it that way, so the firefly-plaid-connector was out for me. I also wanted to implement it in Python and slightly differently, I wanted a sync button inside the transactions page that would fetch the transactions as well as a regularly scheduled sync.
<img width="1971" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/62387276/3be316db-9e09-4014-a249-b15218e5a670">

I'm sharing this here because when I was looking for self-hosted options for financial wellness I landed in this exact discussion looking for options to connect Plaid to Firefly III. @JC5, you and the Firefly III contributors have truly made something spectacular, thank you.

Hopefully if someone stumbles in here looking for ideas or solutions like I did they find what they need.
https://github.com/cskujawa/youteller/tree/main

**Reply by dahifi** [reply]
Dude that's pretty good, will have to check it out. Cross posting over in the issues board: https://github.com/firefly-iii/firefly-iii/issues/7790

**Comment by dvankley** [comment]
Also consider https://github.com/dvankley/firefly-plaid-connector-2
